### DISCUSSION TITLE
How do I use the API getColoredPathway?

### DISCUSSION THREAD

**Discussion by searchlie** [wikipathways/wikipathways-help#85] [Q&A] [answered]
I would like to do something like [KEGG Mapper Color](https://www.genome.jp/kegg/mapper/color.html) with wikipathways.
I searched the wikipathways API and found that getColoredPathway seems to fulfill a similar function.
I created the following Request URL to try it out, but it says "getColoredPathway service is currently not available".
https://webservice.wikipathways.org/getColoredPathway?pwId=WP554&revision=0&graphId=ef1f3&color=00FF00&fileType=png&format=xml
Am I doing something wrong? Or is this API service not currently working?

**Comment by khanspers** [comment, marked as answer]
Hi @searchlie, I believe that method is no longer working. Try the interactive pathway viewer instead:

- Highlight nodes by label and color: `https://pathway-viewer.toolforge.org/embed/WP554?green=ACE`
- Highlight multiple nodes by label and color: `https://pathway-viewer.toolforge.org/embed/WP554?red=ACE&lightgreen=AGT`
- Highlight node by external reference (xref) and color: `https://pathway-viewer.toolforge.org/embed/WP554?red=Ensembl_ENSG00000159640`

**Comment by searchlie** [comment]
@khanspers 
Thank you very much! This is exactly what I was looking for.
Thanks to your tips I was also able to get to [the help page with the proper information](https://www.wikipathways.org/help#analyze).
It seems that [the link I was referring to](https://webservice.wikipathways.org/ui/#!/Download/get_getColoredPathway) was for outdated information.
